private/scripts/sage-theme-install.sh

```diff
@@ -227,24 +226,49 @@ function add_symlink() {
   fi
 
   # Create a files/cache directory on the host.
-  sftp -P 2222 $sftpuser@$sftphost <<EOF
+  sftp -P 2222 "$sftpuser"@"$sftphost" <<EOF
     cd /files
     mkdir cache
 EOF
 
     # Switch back to Git mode.
-    terminus connection:set $sitename.dev git
+    terminus connection:set "$sitename".dev git
 
   # Create the symlink to /files/cache.
-  cd web/app
+  cd web/app || return
   echo -e "${yellow}Adding a symlink to the cache directory.${normal}"
-  ln -sfn uploads/cache
+  ln -sfn uploads/cache .
   git add .
   git commit -m "[Sage Install] Add a symlink for /files/cache to /uploads/cache"
   git push origin master
   cd ../..
 }
 
+# Check if jq is installed. If it's not, try a couple ways of installing it.
+function check_jq() {
+  # Check if jq is installed.
+  if ! command -v jq &> /dev/null; then
+    echo -e "${yellow}jq is not installed.${normal}"
+    # Check if brew is installed.
+    if command -v brew &> /dev/null; then
+      echo -e "${yellow}Installing jq with Homebrew.${normal}"
+      brew install jq
+    # Check if apt-get is installed.
+    elif command -v apt-get &> /dev/null; then
+      echo -e "${yellow}Installing jq with apt-get.${normal}"
+      sudo apt-get install -y jq
+    else
+      echo "${yellow}Attempted to install jq but could not find Brew (for MacOS) or apt-get (for WSL2/Linux). Exiting here. You'll need to add the following lines to your composer.json:${normal}"
+      echo '  "scripts": {'
+      echo '      "post-install-cmd": ['
+      echo "          \"@composer install --no-dev --prefer-dist --ignore-platform-reqs --working-dir=web/app/themes/$sagename\""
+      echo '      ],'
+      echo "${yellow}You might try running 'lando composer install-sage' if you are running locally. Alternately, you can try installing jq another way and then running the script again: https://stedolan.github.io/jq/download/${normal}"
+      exit 1
+    fi
+  fi
+}
```

Comment:
```suggestion
# Check if jq is installed. If it's not, try a couple ways of installing it.
function check_jq() {
  # Check if jq is already installed.
  if command -v jq &> /dev/null; then
      return # jq is already installed
  fi
  echo -e "${yellow}jq is not installed.${normal}"
  # Check if brew is installed and install jq with it
  if command -v brew &> /dev/null; then
    echo -e "${yellow}Installing jq with Homebrew.${normal}"
    brew install jq
    return
  fi
  # Check if apt-get is installed and install jq with it
  if command -v apt-get &> /dev/null; then
    echo -e "${yellow}Installing jq with apt-get.${normal}"
    sudo apt-get install -y jq
    return
  fi
  echo "${yellow}Attempted to install jq but could not find Brew (for MacOS) or apt-get (for WSL2/Linux). Exiting here. You'll need to add the following lines to your composer.json:${normal}"
  echo '  "scripts": {'
  echo '      "post-install-cmd": ['
  echo "          \"@composer install --no-dev --prefer-dist --ignore-platform-reqs --working-dir=web/app/themes/$sagename\""
  echo '      ],'
  echo "${yellow}You might try running 'lando composer install-sage' if you are running locally. Alternately, you can try installing jq another way and then running the script again: https://stedolan.github.io/jq/download/${normal}"
  exit 1
}
```

This is less bash and more how I prefer to avoid nesting as much as possible. feel free to discard.